pkg/api/v1beta1/dynakube/oneagent_types.go

```diff
@@ -60,7 +60,7 @@ type HostInjectSpec struct {
 	// Enabled by default.
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Automatically update Agent",order=13,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
-	AutoUpdate *bool `json:"autoUpdate,omitempty"`
```

Comment:
here its fine, and probably the way to do it, it doesn't make sense to store "nothing"